src/components/datepicker/theme.css

```diff
@@ -82,7 +82,7 @@
 }
 
 .month-picker-field {
-  width: 87px;
+  width: auto !important;
```

Comment:
🤔

Reply:
Would be more helpful to put your thoughts into words here, since I don't know what you're alluring to
So I'll just explain:
As per commit message,
> Fix 100% being inherited from `.wrapper.input`

The `Input` component has general styling on `.wrapper.input` that sets the width to `100%`. In our case, we don't want that; we want that width to be set based on the children's width, so I used `auto`. And since that general `.wrapper.input` styling always overrides, I had to use `!important`

Reply:
Thanks for the explanation! I suppose this could also be fixed by using a more specific input instead of using the abstraction.

Reply:
Sure, but I wouldn't know what would conceptually set it apart from other input types, since it's really specific to our use case, that's why I left it like this until we need a better abtraction